### PR TITLE
No scheduled test

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
-  - 1.22.1
-cache:
-  directories:
-    - $HOME/.pub-cache
+
 dart_task:
-  - test: --platform vm
+  - test
   - dartfmt
   - dartanalyzer
```
Comment (nex3) [inline .travis.yml]: Only run the formatter and analyzer on one Dart version.

File: .travis.yml [hunk 2]
```diff
@@ -1,14 +1,19 @@
 language: dart
-sudo: false
+
 dart:
   - dev
   - stable
   - 1.23.0
```
Comment (nex3) [inline .travis.yml]: Is it worth the resources to test 1.23?

Comment (kevmoo) [reply .travis.yml]: ack

File: .travis.yml (hunk 1 above)
Comment (kevmoo) [reply .travis.yml]: dropped formatter. keeping analyzer – as long as we're supporting 1.x, it's a good sanity check

**Pull request merged**